galaxy_ng/app/api/ui/viewsets/collection.py

```diff
@@ -59,6 +59,9 @@ def get_queryset(self):
             "collection_id",
             "version",
         )
+        deprecations = AnsibleCollectionDeprecated.objects.filter(
+            pk__in=self._distro_content
+        )
```

Comment:
@awcrosby @fao89 this change was needed to make it compatible with latest changes made on pulp_ansible, now Deprecated is a content type, but I need another eyes on this change to confirm I did the right thing.

Before it was

```
deprecated_query = AnsibleCollectionDeprecated.objects.filter(
            collection=OuterRef("collection"), repository_version=self._repository_version
        )
```

Reply:
I believe you need reference something from the main query, maybe:
```suggestion
        deprecations = AnsibleCollectionDeprecated.objects.filter(
            namespace=OuterRef("namespace"), name=OuterRef("name"), pk__in=self._distro_content
        )
```

Reply:
done!